src/main/java/Duke.java

```diff
@@ -1,10 +1,64 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        Task[] list = new Task[100];
+        int listcounter = 0;
+
+        while(true){
```

Comment:
I think this is a coding standard violation. Please refer to the following example from: https://se-education.org/guides/conventions/java/basic.html
E.g:
while (!done) {
    doSomething();
    done = moreToDo();
}